governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Vault Compensation Denomination Poll - April 13, 2020
+summary: Signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+discussion_link: https://forum.makerdao.com/t/1822
+poll_rules: The voter may select to vote for one of the poll options or they may elect to abstain from the poll entirely
+options:
+   0: Abstain
+   1: the asset that was liquidated (ETH or BAT)
+   2: DAI
+
+---
+# Vault Compensation Denomination Poll - April 13, 2020
+
+The Maker Foundation Interim Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the community which the community can use to signal whether to move ahead with a plan to provide compensation to vault holders.
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-scd-faqs/scd-faqs/governance)) will be active for 7 days beginning on Monday, April 13 at 4 PM UTC.
+
+## Review
+
+[An initial poll on whether to compensate vault holders has passed.](https://vote.makerdao.com/polling-proposal/qmwfvvguaf8rz8xwgv2cqnzzt9t5h6epzh17qmk2ue99y4)
+
+When vaults are liquidated, assets are sold for DAI to repay the debt. Any remaining assets are returned to the vault. In brief, in ordinary circumstances, vault holders would receive the type of asset that was originally held in the vault. However, for Maker to provide compensation in the original token, Maker would have to accept the liability of the token appreciating vs DAI prior to when the compensation can be paid out. Do we accept this liability or will we provide compensation in DAI (the DAI value of the residual collateral at the time of liquidation)?
```

Comment:
> Maker would have to accept the liability of the token appreciating vs DAI prior to when the compensation can be paid out

Not sure if I get it right. What's the point of this? Wouldn't compensation in DAI involve token appreciation vs DAI as well?

Reply:
I added an example. Please verify that the math in my example is correct.

Reply:
> If this poll decides tha the compensation

Typo here.
Also 117 should be 118 = (10 ETH x 90 DAI/ETH) - 782 DAI (an explanation where it come from would be helpful).

And anyway both DAI repayment and ETH repayment options don't rely on current DAI/ETH price and in both cases Maker should decide on crash day DAI/ETH price. So what's the point of the liability clause?

Reply:
ETH repayment options does rely on current DAI/ETH price! This ETH would have to be purchased at the current market rate.

Reply:
I updated the example with more clarification. Take a look

Reply:
Ah you're right. Back to original question: what means *liability of the token appreciating*? Isn't it the same as deciding on price of the asset/DAI?

Reply:
If ETH's DAI price goes to moon in the next few weeks, say 1000 DAI/ETH, then that would increase Maker's liability if this poll decides to compensate in ETH units.